src/styles/card.scss

```diff
@@ -0,0 +1,86 @@
+.card {
+  width: $card-width;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  margin: $margin;
```

Comment:
Follow the checklist, please 
![image](https://github.com/mate-academy/layout_catalog/assets/99403198/70895db8-c09f-48a2-ae86-556818d8a16c)

Reply:
Not fixed 
![image](https://github.com/mate-academy/layout_catalog/assets/99403198/c8b65686-0522-4515-a684-632897ef13b4)

Reply:
@MyNovikov you can create an element according to BEM methodology:
```
<div class="card container__card">
```
And use margin rules for the `container__card` selector.